app/code/Magento/WishlistGraphQl/etc/schema.graphqls

```diff
@@ -5,6 +5,19 @@ type Query {
     wishlist: WishlistOutput @resolver(class: "\\Magento\\WishlistGraphQl\\Model\\Resolver\\WishlistResolver") @doc(description: "The wishlist query returns the contents of a customer's wish list") @cache(cacheable: false)
 }
 
+type Mutation {
+    addItemsToWishlist (input: AddItemsToWishlistInput): AddItemToWishlistOutput @resolver(class: "\\Magento\\WishlistGraphQl\\Model\\Resolver\\AddItemsToWishlist") @doc(description:"Add items wishlist")
+}
+
+input AddItemsToWishlistInput {
+    wishlist_items: [WishlistItemInput!]!
+}
+
+input WishlistItemInput {
+    sku: String!
+    quantity: Float
```

Comment:
This will work only for some product types, we need to design GraphQL schema for all product types (including configurable, bundle, grouped). Custom options will also not be supported.

Since there is a way to convert wishlist item to cart item, the interface of adding item to wishlist should be similar to the interface for adding items to cart. 

The schema must be reviewed and approved by @mbrinton01 (wishlist product owner) and @paliarush (wishlist architect) before implementation.